src/gapstate.h

```diff
@@ -65,27 +66,10 @@ typedef struct GAPState {
     UInt   NrError;
     UInt   NrErrLine;
     UInt   Symbol;
+
     const Char * Prompt;
-#if defined(HPCGAP)
-    TypInputFile *  InputFiles[16];
-    TypOutputFile * OutputFiles[16];
-    int             InputFilesSP;
-    int             OutputFilesSP;
-#else
-    TypInputFile  InputFiles[16];
-    TypOutputFile OutputFiles[16];
-#endif
-    TypInputFile *  Input;
-    Char *          In;
-    TypOutputFile * Output;
-    TypOutputFile * InputLog;
-    TypOutputFile * OutputLog;
-    TypOutputFile * IgnoreStdoutErrout;
-    TypOutputFile   InputLogFileOrStream;
-    TypOutputFile   OutputLogFileOrStream;
-    Int             NoSplitLine;
-    Char            Pushback;
-    Char *          RealIn;
+
+    Char * In;
```

Comment:
I am leaving `In` inside `GAPState` for now, because to me it's not all that clear where it "really" belongs. Maybe into `io.c`, but maybe also into `scanner.c` -- or perhaps rather a new `linereader.c` which sits between the two? Or a third place?